radiometric_normalization/gimage.py

```diff
@@ -173,3 +173,38 @@ def check_comparable(gimages, check_metadata=False):
             raise Exception(
                 'Image {} has different geographic metadata: {} '
                 '(initial: {})'.format(i + 1, image.metadata, metadata))
+
+
+def check_equal(gimages, check_metadata=False):
+    '''Checks that the gimages are equavalent (returns a boolean)'''
+
+    all_equal = True
+    try:
+        check_comparable(gimages, check_metadata)
+    except:
+        all_equal = False
+
+    if all_equal is True:
```

Comment:
This could be cleaner if you use exception handling vs checking if all_equal is True multiple times. For the exception route, you could create a fcn that raises an exception for any case where you would set all_equal=False. You can either allow the exception to be raised (matches functionality of check_comparable) or catch it and return False (if you want to stick with Boolean return)